gradle.properties

```diff
@@ -8,15 +8,22 @@ toolkitVersion=1.26-SNAPSHOT
 publishToken=
 publishChannel=
 
-# Common dependencies
 ideProfileName=2020.2
+
+# Common dependencies
 kotlinVersion=1.4.21
 awsSdkVersion=2.16.3
 coroutinesVersion=1.3.3
-ideaPluginVersion=0.7.2
 ktlintVersion=0.39.0
 jacksonVersion=2.11.1
 telemetryVersion=0.0.91
+commonsMarkVersion=0.17.1
+jgitVersion=5.11.0.202103091610-r
+
+gradleRetryPluginVersion=1.2.1
+gradleTestLoggerPlugin=2.1.1
+ideaPluginVersion=0.7.2
+ktintPluginVersion=9.4.1
```

Comment:
Can you add a comment that we need to change the plugin and not to update this one for now?

Reply:
Added